### ISSUE TITLE
FEW questions on models and PCK metric

### ISSUE THREAD

**Issue by Dipankar1997161** [david8862/tf-keras-stacked-hourglass-keypoint-detection#18]
Hey @david8862 
**Question 1:**
I trained my model and I even got the keypoint_results.json in the results directory. However

**My model was not saved. I could not find it anywhere. Why is this happening?**

I just trained it under 1 epoch for testing reasons before I can train it for long.

**Question 2:**
Why does the file "Eval.py" run the very next moment the training is finished for 1 epoch? 

I mean if I just get the model for train.py after running 10 epochs.  Won't that be better to evaluate then rather than evaluate after every epoch.?

**Question 3:**
after my evaluation was done, my program terminated with a **ZeroDivisionError**  which prevented it from creating the PCK metric image and further steps. The line of code where it happened seems to be right after **json.dumps(output_list)**. However, the keypoint_results.json file was generated successfully!

![ZeroDivisionError](https://user-images.githubusercontent.com/85514219/166118483-b1aa6338-ef28-49d8-acb0-c9f67938a9fc.png)

**Question 4:**
I ran my training for 1 epoch and I find the valid.txt empty. Just the name of the model with input size. Is this natural?

![valid](https://user-images.githubusercontent.com/85514219/166118663-882f9396-2ecd-44e1-8a32-d5bca3a5bd55.png)

**It will be great if you can share a snippet of how the val.txt would look. Just to get an overview!**

I know these are too many questions, but since I am almost at the end of the project, I want to be clarified!!

Looking to hearing from you!
Thank you once again @david8862
